### PR TITLE
Licensed features docs

### LINKED ISSUE #3911: Add docs for managing license keys
We need documentation that explains how to add a license key to Coder deployment and what this unlocks (currently just audit logs).

### PR DESCRIPTION
**Pull request by spikecurtis**
Fixes #3911

### CODE REVIEW

File: docs/manifest.json [hunk 1]
```diff
@@ -39,6 +39,11 @@
           "title": "Upgrading",
           "description": "Learn how to upgrade Coder.",
           "path": "./install/upgrade.md"
+        },
+        {
+          "title": "Licensed Features",
```
Comment (ammario) [inline docs/manifest.json:44]: All the features are licensed, technically. I think it would be more clear to title this page `Enterprise`.

**Review by ammario — CHANGES_REQUESTED** [review]

File: docs/manifest.json (hunk 1 above)
Comment (spikecurtis) [reply docs/manifest.json:44]: And also "learn how to enable Enterprise features." ?

Comment (ammario) [reply docs/manifest.json:44]: Yeah I think that makes sense.

**Review by ammario — APPROVED** [review]
Apologies for the late review

**Review by bpmct — APPROVED** [review]
Suggest moving this under the "Admin" section now

**Pull request merged**